examples/stacking_analysis/Example_fit_halo_mass_stacking_analysis.ipynb

```diff
@@ -0,0 +1,881 @@
+    "        \n",
+    "        cl_cut_jack = clmm.GalaxyCluster('Stack', single_catalog.ra, single_catalog.dec, single_catalog.z, single_catalog.galcat[mask_out])\n",
+    "        \n",
+    "        ds, r, sum_weights = excess_surface_density(single_catalog = cl_cut_jack, radial_bin = radial_bin, sigma_c = 'sigma_c')\n",
+    "        \n",
+    "        gt.append(ds)\n",
+    "\n",
+    "    gt = np.array(gt)\n",
+    "    \n",
+    "    cov = np.cov(gt.T)*(N_jk-1)**2/N_jk\n",
+    "    \n",
+    "    plt.xlabel('ra')\n",
+    "    plt.ylabel('dec')\n",
+    "    plt.axis('equal')\n",
+    "        \n",
+    "    return cov"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Jackknife covariance matrix\n",
+    "cov_single = jackknife(single_catalog = cl_single, N_jk = 100)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig, axs = plt.subplots(2,2, figsize = (20,10))\n",
+    "#axs[0,0].imshow(np.corrcoef(cov_ss), vmin = -1,, cmap = 'bwr', origin = 'lower')\n",
+    "min_value = min(sample_covariance_stack_true.flatten())\n",
+    "im = axs[0,0].imshow(cov_single, cmap = 'Reds', origin = 'lower', label = 'single-jackknife')\n",
+    "plt.colorbar(im, ax=axs[0,0])\n",
+    "axs[0,1].errorbar(r_single,ds_single, np.sqrt(cov_single.diagonal()), label = 'single')\n",
+    "moo.set_concentration(concentration.mean())\n",
+    "moo.set_mass(cluster_m.mean())\n",
+    "axs[0,1].plot(r_single,moo.eval_excess_surface_density(r_single, cluster_z.mean()), label = 'fiducial')\n",
+    "axs[0,1].set_ylim(0.3*1e13, 3.5*1e13)\n",
+    "\n",
+    "im = axs[1,0].imshow(sample_covariance_stack_true, cmap = 'Reds', origin = 'lower' ,label = 'stack-sample')\n",
+    "plt.colorbar(im, ax=axs[1,0])\n",
+    "\n",
+    "plt.figure(figsize = (20,10))\n",
+    "for i in range(n_catalogs):\n",
+    "    axs[1,1].plot(Stack_file_true['r_single'][i], Stack_file_true['ds_single'][i], alpha = 0.01, c = 'grey', linewidth =20)\n",
+    "axs[1,1].errorbar(r_mean_true,ds_mean_true, np.sqrt(sample_covariance_stack_true.diagonal()), linewidth = 3, label = 'stack - true z')\n",
+    "axs[1,1].errorbar(r_mean_photoz,ds_mean_photoz, np.sqrt(sample_covariance_stack_photoz.diagonal()), linewidth = 3,fmt = '.', label = 'stack - photoz')\n",
+    "axs[1,1].set_ylim(0.3*1e13, 3.5*1e13)\n",
+    "\n",
+    "axs[0,0].set_title('covariance matrix', fontsize = 20)\n",
+    "axs[0,1].set_title('Excess surface density profile', fontsize = 20)\n",
+    "for i in range(2):\n",
+    "    axs[i,1].set_xlim(min(r_single), max(r_single))\n",
+    "    axs[i,1].set_ylim(0.3*1e13, 3.5*1e13)\n",
+    "    axs[i,1].set_xlabel('R [Mpc]', fontsize = 20)\n",
+    "    axs[i,1].set_ylabel(r'$\\Delta\\Sigma$', fontsize = 20)\n",
+    "    \n",
+    "for i in range(2):\n",
+    "    for j in range(2):\n",
```

Comment:
Get warnings and cannot see legends in the covariance matrix images. Would it be better to only show legends for axs with j=1?